omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -607,6 +608,7 @@
 default['private_chef']['postgresql']['log_rotation']['num_to_keep'] = 10
 default['private_chef']['postgresql']['username'] = "opscode-pgsql"
 default['private_chef']['postgresql']['db_superuser'] = 'opscode-pgsql'
+default['private_chef']['postgresql']['db_connection_superuser'] = nil
```

Comment:
Since the usage doesn't make clear why the attribute is necessary, could you add a comment here explaining the why and how of these attributes, and  a one-liner[1]  at the other other new  `*connection_user` additions?

[1] something along the lines of `See ['postgresql']['db_connection_superuser'] for information`